tests/fixtures/fixtures_asset.py

```diff
@@ -155,7 +157,7 @@ def asset_mock2(auth_mock, requests_mock):
     )
     asset = Asset(auth=auth_mock, asset_id=ASSET_ID2)
     return asset
-
+    
```

Comment:
Always happens to me, unwanted tabs.

Reply:
ahh, let me apply black formatting.
We should also configure pre-commit for SDK. 
I will create a ticket in the backlog